bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/bookie/SanityTestCommand.java

```diff
@@ -134,8 +134,7 @@ public static CompletableFuture<Boolean> handleAsync(ServerConfiguration conf, S
             }
             CompletableFuture<LedgerHandle> lhFuture = new CompletableFuture<>();
             CompletableFuture<Void> readEntryFuture = new CompletableFuture<>();
-
-            FutureUtils.collect(entriesFutures).thenCompose(_r -> {
+            FutureUtils.collect(entriesFutures).thenCompose(_r -> lh.closeAsync()).thenCompose(_r -> {
```

Comment:
It seems the `lh` still in use at the next CompletionStage. Why close it here? It will close at the end.
```
                    lhFuture.complete(lh);
                }, null);
                return lhFuture;
            }).thenCompose(rlh -> {
```

Reply:
We can close the ledgerHandle to avoid ledger recovery, or we will see the ledger fence in the bookie server.

After writing done, we can close the ledger handle.